packaging/resource_suite_s3_nocache.py

```diff
@@ -2277,6 +2132,195 @@ def test_s3_in_replication_node__issue_2102(self):
             self.admin.assert_icommand("iadmin rmresc %s" % s3_resc2_resource_name)
             self.admin.assert_icommand("iadmin rmresc %s" % s3_resc3_resource_name)
 
+# The tests in this class take a long time and do not need to run in every different test suite
+class Test_S3_NoCache_Large_File_Tests_Base(Test_S3_NoCache_Base):
```

Comment:
The tests in red above are the ones you moved into this class, correct?

Reply:
Yes, there are three tests that were moved.

Reply:
Cool. You can mark this as resolved.